Makefile.common

```diff
@@ -59,7 +55,7 @@ ifeq (,$(findstring $(strip $(target)),$(known_targets)))
   target := Default
 endif
 
-$(info Will build using target: $(target) (detected os: $(OS_NAME), arch: $(OS_ARCH)))
+$(info Will build using target: $(target) (detected os: $(OS_NAME) arch: $(OS_ARCH)))
```

Comment:
why the change here?

Reply:
I reverted the Makefile and Makefile.common changes (mostly adding slf4j to the build) because they broke the "amalgamated" job in CI. I assume the slf4j jar is needed for compiling some Java things from the Makefile.

Reply:
reverting is fine, we shouldn't need slf4j anymore in the makefile if it's optional, i was more referring to that particular `,` 🙃

Reply:
The changes I made there were themselves reverting the addition of slf4j jar from previous commits. The comma seems to be present in that commit: https://github.com/xerial/sqlite-jdbc/commit/1663cfec7a7bf167f299f5b93f0af754649e6c82#diff-f190d29e2cad4f32539c8b73d8af07c77f0aac63b5c6553c06b117c08a70b494R62

I am not sure what to say about the slf4j dependency in Makefile and Makefile.common; without it present, the "amalgamation" job fails to `javac` successfully.

Reply:
Note that this change is moot since I reverted to the old Makefile.common (which does have the comma, added in 1663cfe).